### PR TITLE
MK, RK, Pacific Ports

### PR DESCRIPTION
**Pull request by tylerthetiletiler**
Mackay Aerodrome
Rocky Aerodrome

Made a start on Pacific Ports

Removed VATNZ material

Amended TODO

### CODE REVIEW

File: docs/aerodromes/Mackay.md [hunk 1]
```diff
@@ -4,4 +4,44 @@
...
+The Standard Assignable level from MK TWR to COR TCU is the lower of `A050` or the `RFL`, any other level must be prior coordinated.
+
+### Arrivals
+COR TCU will coordinate the sequence to MK TWR
+
+!!! example
+    **COR** -> **MK TWR**: "New Sequence of 2. Via LEDUS, VOZ1164, Number 1. JST453, Number 2”  
+    **MK TWR** -> **COR**: "VOZ1164, Number 1. JST453, Number 2"  
+
+The Standard Assignable level from COR TCU to MK TWR is `A060`, any other level must be prior coordinated.
```
Comment (alexd-332) [inline docs/aerodromes/Mackay.md:47]: No standard assignable, just cleared for approach and transferred to tower prior to 10nm, or overhead the aid if doing VOR/NDB reversal

File: docs/aerodromes/Rockhampton.md [hunk 2]
```diff
@@ -4,4 +4,44 @@
...
\ No newline at end of file
+## Positions
+
+| Name | Callsign | Frequency | Login Identifier |
+| ---- | -------- | --------- | ---------------- |
+| Rockhampton ADC | Rocky Tower | 118.100 | RK_TWR |
+| Rockhampton SMC | Rocky Ground | 121.800 | RK_GND |
+| Rockhampton ATIS |    | 128.500 | YBRK_ATIS |
+
+## Airspace
```
Comment (alexd-332) [inline docs/aerodromes/Rockhampton.md:15]: Note about RK TWR owning RK CTR SFC-A010

File: docs/aerodromes/Rockhampton.md [hunk 3]
```diff
@@ -4,4 +4,44 @@
...
+The Standard Assignable level from RK TWR to COR TCU is the lower of `A050` or the `RFL`, any other level must be prior coordinated.
+
+### Arrivals
+COR TCU will coordinate the sequence to RK TWR
+
+!!! example
+    **COR** -> **RK TWR**: "New Sequence of 2. Via SARUS, VJN, Number 1. WFE, Number 2”  
+    **RK TWR** -> **COR**: "VJN, Number 1. WFE, Number 2"  
+
+The Standard Assignable level from COR TCU to MK TWR is `A060`, any other level must be prior coordinated.
```
Comment (alexd-332) [inline docs/aerodromes/Rockhampton.md:47]: No standard assignable, just cleared for approach and transferred to tower prior 10nm, or overhead the aid if doing VOR/NDB reversal

**Review by alexd-332 — CHANGES_REQUESTED** [review]
Also apologies, Coral TCU owns up to F150, not F155 (finally found doc)

**Review by alexd-332 — DISMISSED** [review]

**Review by alexd-332 — APPROVED** [review]

**Review by zkgell — APPROVED** [review]
changes not required

**Review by mattkelly4 — APPROVED** [review]

**Pull request merged**